src/components/iso4217.service.js

```diff
@@ -4,868 +4,876 @@
+      },
+      'TJS': {
+        text: 'Somoni',
+        fraction: 2,
+        symbol: false
+      },
+      'TZS': {
+        text: 'Tanzanian Shilling',
+        fraction: 2,
+        symbol: false
+      },
+      'THB': {
+        text: 'Baht',
+        fraction: 2,
+        symbol: '฿'
+      },
+      'TOP': {
+        text: 'Pa’anga',
+        fraction: 2,
+        symbol: false
+      },
+      'TTD': {
+        text: 'Trinidad nd Tobago Dollar',
+        fraction: 2,
+        symbol: 'TT$'
+      },
+      'TND': {
+        text: 'Tunisian Dinar',
+        fraction: 3,
+        symbol: false
+      },
+      'TRY': {
+        text: 'Turkish Lira',
+        fraction: 2,
+        symbol: ''
+      },
+      'TMT': {
+        text: 'Turkmenistan New Manat',
+        fraction: 2,
+        symbol: false
+      },
+      'UGX': {
+        text: 'Ugand Shilling',
+        fraction: 0,
+        symbol: false
+      },
+      'UAH': {
+        text: 'Hryvnia',
+        fraction: 2,
+        symbol: '₴'
+      },
+      'AED': {
+        text: 'UAE Dirham',
+        fraction: 2,
+        symbol: false
+      },
+      'USN': {
+        text: 'US Dollar (Next day)',
+        fraction: 2,
+        symbol: false
+      },
+      'UYI': {
+        text: 'Uruguay Peso en Unidades Indexadas (URUIURUI)',
+        fraction: 0,
+        symbol: false
+      },
+      'UYU': {
+        text: 'Peso Uruguayo',
+        fraction: 2,
+        symbol: '$U'
+      },
+      'UZS': {
+        text: 'Uzbekistan Sum',
+        fraction: 2,
+        symbol: 'лв'
+      },
+      'VUV': {
+        text: 'Vatu',
+        fraction: 0,
+        symbol: false
+      },
+      'VEF': {
+        text: 'Bolivar',
+        fraction: 2,
+        symbol: 'Bs'
+      },
+      'VND': {
+        text: 'Dong',
+        fraction: 0,
+        symbol: '₫'
+      },
+      'YER': {
+        text: 'Yemeni Rial',
+        fraction: 2,
+        symbol: '﷼'
+      },
+      'ZMW': {
+        text: 'Zambian Kwacha',
+        fraction: 2,
+        symbol: false
+      },
+      'ZWL': {
+        text: 'Zimbabwe Dollar',
+        fraction: 2,
+        symbol: false
+      }
+    };
 
-			/**
-			 * retrieves the object holding currency, code and fraction information about a currency.
-			 * 
-			 * @param string code
-			 * @return object
-			 */
-			getCurrencyByCode: function(code) {
-				if (!code) return;
+    return {
+      /**
+       * retrieves the object holding currency, code and fraction information about a currency.
+       *
+       * @param string code
+       * @return object
+       */
+      getCurrencyByCode: function(code) {
+        if (!code) return;
 
-				return currencies[code.toUpperCase()];
-			}
-		};
-	});
+        return currencies[code.toUpperCase()];
+      },
+      /**
+       * retrieves the internal list of currencies, as is.
+       *
+       * @return object
+       */
+      getCurrenciesList: function() {
+        return currencies;
+      }
```

Comment:
This is the bit that was added (commenting on it since re-indentation marks the whole file as modified...)